ghost/i18n/locales/hu v2/ghost.json

```diff
@@ -0,0 +1,34 @@
+{
+    "All the best!": "Üdvözlettel,",
+    "Complete signup for {{siteTitle}}!": "{{siteTitle}} —\u00a0Erősítsd meg a feliratkozásodat a hírlevélre!",
```

Comment:
Not all Ghost sites have a newsletter.  Some take sign-ups for web-only access.  This language needs adjustment.